dashboard/src/components/AppView/AppControls/RollbackButton/RollbackDialog.tsx

```diff
@@ -53,7 +54,18 @@ function RollbackDialog({
         <CdsModal closable={false} onCloseChange={closeModal}>
           <CdsModalHeader>Rollback application</CdsModalHeader>
           <CdsModalContent>
-            {error && <Alert theme="danger">An error occurred: {error.message}</Alert>}
+            {error &&
+              (error.constructor === FetchWarning ? (
+                <Alert theme="warning">
+                  There is a problem with this package: {error["message"]}
```

Comment:
Added `FetchWarning` to allow viewing partially-missing packages and degraded interactions (ie, just delete the package, not deleting or rolling back)